php-fpm-healthcheck

```diff
@@ -56,7 +56,11 @@ get_fpm_status() {
     
     # Since I cannot use pipefail I'll just split these in two commands
     FPM_STATUS=$(env -i REQUEST_METHOD="$REQUEST_METHOD" SCRIPT_NAME="$SCRIPT_NAME" SCRIPT_FILENAME="$SCRIPT_FILENAME" "$FCGI_CMD_PATH" -bind -connect "$1" 2> /dev/null)
-    FPM_STATUS=$(echo "$FPM_STATUS" | tail +5)
+    if [ -f /etc/redhat-release ]; then
+        FPM_STATUS=$(echo "$FPM_STATUS" | tail -n +5)
```

Comment:
Thinking about this, we can always use `-n` because it is POSIX-compliant.

Can you change your commit adding only this parameter?

```diff
diff --git a/php-fpm-healthcheck b/php-fpm-healthcheck
index b476190..cd69742 100755
--- a/php-fpm-healthcheck
+++ b/php-fpm-healthcheck
@@ -56,7 +56,7 @@ get_fpm_status() {
     
     # Since I cannot use pipefail I'll just split these in two commands
     FPM_STATUS=$(env -i REQUEST_METHOD="$REQUEST_METHOD" SCRIPT_NAME="$SCRIPT_NAME" SCRIPT_FILENAME="$SCRIPT_FILENAME" "$FCGI_CMD_PATH" -bind -connect "$1" 2> /dev/null)
-    FPM_STATUS=$(echo "$FPM_STATUS" | tail +5)
+    FPM_STATUS=$(echo "$FPM_STATUS" | tail -n +5)
 
     if test "$VERBOSE" = 1; then printf "php-fpm status output:\\n%s\\n" "$FPM_STATUS"; fi;
 
```

Reply:
Did the update, like this?

Reply:
Yes! Thank you :heart: